kubernetes-templates/04_cvat_backend_deployment.yml

```diff
@@ -0,0 +1,96 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cvat-backend
+  namespace: cvat
+  labels:
+    app: cvat-app
+    tier: backend
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: cvat-app
+      tier: backend
+  template:
+    metadata:
+      labels:
+        app: cvat-app
+        tier: backend
+    spec:
+      containers:
+        - name: cvat-backend-app-container
+          image: openvino/cvat_server:v1.2.0
+          imagePullPolicy: Always
+          resources:
+            requests:
+              cpu: 10m
+              memory: 100Mi
+          env:
+          - name: DJANGO_MODWSGI_EXTRA_ARGS
+            value: ""
+          - name: UI_PORT
+            value: "80"
+          - name: UI_HOST
+            value: "cvat-frontend-service"
+          - name: ALLOWED_HOSTS
+            value: "{MY_SERVER_URL_COM}"
```

Comment:
Currently, the readme file does not mention this parameter, so we will set the `*` here as in the compose config: https://github.com/openvinotoolkit/cvat/blob/develop/docker-compose.yml#L42.
or please note about that in Readme.md

Reply:
Adjusted in 590d7f09